src/sentry_plugins/pagerduty/plugin.py

```diff
@@ -49,7 +49,10 @@ def notify_users(self, group, event, fail_silently=False, **kwargs):
         if not self.is_configured(group.project):
             return
 
-        description = event.get_legacy_message()[:1024]
+        # TODO: This should eventually be event.title in line with other plugins.
+        # However, we should notify users first, since PD alert routing may be
+        # based off the message field.
+        description = (event.real_message or event.message)[:1024]
```

Comment:
What's the difference in content between the logentry attribute you were using before and real_message/event.message

Reply:
There shouldn't be any difference in content here, this is just a refactor to clean up the event class and move the Pagerduty specific message into the plugin

Reply:
Could you please expand on the "shouldn't be any difference" ? You are now fetching a different field from the event. Where was the old `logentry` populated from ?

Reply:
I mean if you look at the implementation of `real_message` vs `get_legacy_message()`, they are almost exactly the same except that legacy message falls back to `event.message`, which is what we have done here.

Reply:
damn you are right, I was looking at the wrong method.